webapp/src/components/link_tooltip/tooltip.css

```diff
@@ -115,8 +115,10 @@
     padding: 0 5px;
     font: .75em/2 SFMono-Regular,Consolas,Liberation Mono,Menlo,monospace;
     color: var(--blue);
-    white-space: nowrap;
     background-color: var(--light-blue);
     border-radius: 3px;
-    maxWidth: 140px;
+    max-width: 140px;
+    text-overflow: ellipsis;
+    white-space: nowrap;
```

Comment:
Any strong reason to move `white-space`? If not, maybe move it back to where it was, to have a cleaner diff.

Reply:
I like (1/5) having all rules related to the size and behaviour of the box grouped together.